GitExtUtils/GitUI/UIExtensions.cs

```diff
@@ -43,5 +45,27 @@ public static bool IsFixedWidth(this Font ft, Graphics g)
 
             return fixedWidth;
         }
+
+        public static string? FormatBodyAndNotes(string? body, string? notes)
+        {
+            if (string.IsNullOrWhiteSpace(notes))
+            {
+                return body;
+            }
+
+            const string notesPrefix = "Notes:";
+            const string indent = "    ";
+            StringBuilder? sb = new StringBuilder((body?.Length ?? 0) + 2 + notesPrefix.Length + 2 + indent.Length + notes.Length + 1)
```

Comment:
ValueStringBuilder or Span<> would be more effective - assuming the stack is big enough and lines can be estimated cheaply.
Not requesting changes, this should not be performance critical.